templates/github/.github/workflows/scripts/install.sh.j2

```diff
@@ -25,7 +25,9 @@ fi
 
 pip install ${PIP_REQUIREMENTS[*]}
 
+pulp config create --base-url {{ pulp_scheme }}://pulp{% if pulp_scheme != 'https' %} --no-verify-ssl{% endif %} --api-root "$PULP_API_ROOT" --username "admin" --password "password"
 {% if test_cli -%}
+cp ~/.config/pulp/cli.toml "${REPO_ROOT}/../{{ cli_package }}/tests/cli.toml"
 PULP_CLI_VERSION="$(pip freeze | sed -n -e 's/{{ cli_package }}==//p')"
 git clone --depth 1 --branch "$PULP_CLI_VERSION" {{ cli_repo }} ../{{ cli_package }}
 {%- endif %}
```

Comment:
Hmm, I've just reviewed that move, around line 138 we have:

```
  {%- if test_reroute %}
  export PULP_API_ROOT="/rerouted/djnd/"
  {%- endif %}
```

So I guess it really should be down there, so it can use the proper value of PULP_API_ROOT.

Reply:
Hmm. that explains why we do it down there.
Let's not do this now then. But we should think about a more structured approach to set all this up.

Reply:
Reverted